docs/rest_api.md

```diff
@@ -416,6 +416,31 @@ Authorization requirements:
 Power cycle the node named `<node>`, and set it's next boot device to
 PXE. If the node is powered off, this turns it on.
 
+#### node_set_bootdev
+
+`POST /node/<node>/dev/<dev>/set_bootdev`
+
+Sets the nodes next boot device persistently
+
+device <dev> can be set to:
```

Comment:
This should probably be a bullet list, e.g:

* "none": Do not change boot device order
* "disk": Force boot from default Hard-drive
* ... and so on

Worth famliarizing yourself with markdown formatting rules, if you haven't already.